shop/src/main/resources/db/migration/V1_0__create_tables.sql

```diff
@@ -0,0 +1,74 @@
+CREATE TABLE IF NOT EXISTS product_category (
+                                                id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+                                                name NVARCHAR(100) NOT NULL,
+                                                description NVARCHAR(100) NOT NULL
+);
+CREATE TABLE IF NOT EXISTS supplier (
+                                        id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+                                        name NVARCHAR(100) NOT NULL
+);
+CREATE TABLE IF NOT EXISTS customer (
+                                        id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+                                        first_name NVARCHAR(100) NOT NULL,
+                                        last_name NVARCHAR(100) NOT NULL,
+                                        username NVARCHAR(100) NOT NULL,
+                                        password NVARCHAR(100) NOT NULL,
+                                        email NVARCHAR(100) NOT NULL
+);
+CREATE TABLE IF NOT EXISTS product (
+                                       id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+                                       name NVARCHAR(100) NOT NULL,
+                                       description NVARCHAR(255) NOT NULL,
+                                       price NUMERIC NOT NULL,
+                                       weight DOUBLE PRECISION NOT NULL,
+                                       category_id INTEGER NOT NULL REFERENCES product_category(id),
+                                       supplier_id INTEGER NOT NULL REFERENCES supplier(id),
+                                       image_url NVARCHAR(255) NOT NULL
+);
+
+create table if not exists location(
```

Comment:
dont change case half way 😄 
There is a shortcut in intellij for a simpler life with upper and lower case: CTRL+SHIFT + U